docs/docs/deploying-to-azure.md

```diff
@@ -0,0 +1,136 @@
+   | _Resource Group_ | _A pre-existing or new resource group_ |
+
+1. Enter the **Static Web Apps details**.
+
+   | Setting  | Value                                                                         |
+   | -------- | ----------------------------------------------------------------------------- |
+   | _Name_   | Name your app. Valid characters are `a-z` (case insensitive), `0-9`, and `_`. |
+   | _Region_ | Select Region closest to you                                                  |
+   | _SKU_    | **Free**                                                                      |
+
+1. Click the **Sign-in with GitHub** button and authenticate with GitHub.
+1. Enter the **Source Control Details**.
+
+   | Setting        | Value                                                    |
+   | -------------- | -------------------------------------------------------- |
+   | _Organization_ | Select the Organization where you created the repository |
+   | _Repository_   | **gatsby-app**                                           |
+   | _Branch_       | **master**                                               |
+
+1. Click the **Next: Build >** button to edit the build configuration.
+
+### Building your site
+
+Next, add configuration details specific to your preferred frontend framework.
+
+| Setting                 | Value         |
+| ----------------------- | ------------- |
+| _App location_          | _Leave blank_ |
+| _Api location_          | _Leave blank_ |
+| _App artifact location_ | **build**     |
+
+Click the **Review + create** button.
+
+Continue to create the application.
+
+1. Click the **Create** button
+
+1. Once the deployment is complete, click the **Go to resource** button
+
+### Review the GitHub Action
+
+At this stage, your Static Web Apps instance is created in Azure but your app is not yet deployed. The GitHub Action that Azure creates in your repository will run automatically to perform the first build and deployment of your app, but it takes a couple of minutes to finish.
+
+You can check the status of your build and deploy action by clicking a big blue border at the top of your main window.
+
+### View your website
+
+Once your GitHub Action finishes building and publishing your web app, you can browse to see your running app.
+
+Click on the _URL_ link in the Azure portal to visit your app in the browser.
+
+Congratulations! You've deployed your first app to Azure Static Web Apps!
+
+> Don't worry if you see a web page that says the app hasn't been built and deployed yet. Try refreshing the browser in a minute. The GitHub Action runs automatically when the Azure Static Web Apps is created. So if you see the splash page, the app is still being deployed.
+
+## Additional resources
+
+There's much more to learn about Azure Static Web Apps such as working with routes, setting up custom domains, adding a Serverless API, and much more. Below are some useful links:
```

Comment:
These are things which could potentially be added to this reference guide in the future!

Reply:
These are an important part for people to learn more about the service. Deploying is just the beginning. :)

Reply:
but if you mean adding written sub-sections on what the links are pointing to then I definitely agree it could be a great thing to add in the future :)

Reply:
^ that is exactly what I mean! There's room for this guide to expand. :D